scripts/prepare_eez_data/prepare_eez_data.py

```diff
@@ -0,0 +1,108 @@
+import geopandas as gpd
+import json
+import argparse
+import logging
+import pandas as pd
+
+# Configure logging
+logging.basicConfig(format='%(levelname)s: %(message)s', level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+
+def load_csv(csv_file):
+    try:
+        csv_data = pd.read_csv(csv_file)
+        return csv_data
+    except FileNotFoundError:
+        logger.error(f"CSV file '{csv_file}' not found.")
```

Comment:
```suggestion
        logger.error(f"{csv_file = } not found.")
```
Just because it's a bit of python sugar I love :) Prints e.g. `"csv_file = 'my_file.csv' not found."`